ios/RNGoogleMobileAds/RNGoogleMobileAdsAppOpenModule.mm

```diff
@@ -0,0 +1,96 @@
+/**
+ * Copyright (c) 2016-present Invertase Limited & Contributors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this library except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+#import "RNGoogleMobileAdsAppOpenModule.h"
```

Comment:
I won't comment on every file everywhere but basically, before any RNGoogleMobile... or UMP... or GoogleMobile... anything symbol is touched, in each .h and .mm, have to do the macCatalyst ifdef, then endif way down at the end